test/GenerisTestCase.php

```diff
@@ -59,18 +68,36 @@ protected function getOntologyMock()
             'smoothcache' => new \common_cache_NoCache()
         ]);
         $session->setServiceLocator($sl);
-        $model = new \core_kernel_persistence_smoothsql_SmoothModel([
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_PERSISTENCE => 'mockSql',
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_READABLE_MODELS=> [123],
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_WRITEABLE_MODELS=> [123],
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_NEW_TRIPLE_MODEL=> 123,
-            'cache' => 'smoothcache'
-        ]);
         $model->setServiceLocator($sl);
 
         return $model;
     }
 
+    /**
+     * @return FileSystemService
+     */
+    protected function getFileSystemMock($dirs = [])
+    {
+        $adapterparam = [
+            'testfs' => class_exists('League\Flysystem\Memory\MemoryAdapter')
```

Comment:
Maybe it would be better to add `league/flysystem-memory` dependency in `require-dev` section in composer.json instead of checking if class exists?